openlibrary/templates/type/author/view.html

```diff
@@ -170,6 +170,16 @@ <h6 class="collapse black uppercase">$label</h6>
             $:render_subjects(_("Time"), books.facet_counts.get('time_facet'), 'time:')
         <!-- /SUBJECTS -->
 
+        <div class="section">
+            <h6 class="collapse black uppercase">TESTING ONLY WIKIDATA SECTION</h6>
+            <p style="text-align: center;">
+                <!-- This is only below subject temporarily to avoid merge conflicts on testing -->
+                $ wikidata = page.wikidata(fetch_missing=show_librarian_extras)
+                $if wikidata:
+                    $wikidata.get_description(i18n.get_locale())
+            </p>
+        </div>
+
```

Comment:
I am leaving this code here now for testing.openlibrary.org to be easier to verify this PR on.

I can either remove it before we merge this or there is already a commit to remove it in This is removed in https://github.com/internetarchive/openlibrary/pull/9130